src/components/Controls.vue

```diff
@@ -2,7 +2,21 @@
   <div class="field is-grouped">
     <p class="control">
       <label class="label">Activators</label>
-      <input v-model="activators" @input="updateActivators" class="input is-rounded is-small" type="text">
+      <input
+        v-model="activators"
```

Comment:
@navyhockey56 This `v-model` isn't actually doing anything, it was dead code of mine left on master...sorry!